testPI2/integration/helpers/testProducer.js

```diff
@@ -44,7 +62,7 @@ const messageProtocol = {
   metadata: {
     event: {
       id: Uuid(),
-      type: 'prepare',
+      type: 'transfer',
```

Comment:
1.  Should this not be 'prepare'?
2. Should we not create a some new objects/methods to assist in testing the fulfil messages instead of changing the existing one?

Reply:
2. I agree. I created the requested objects and methods and I've made a step further adding another handler, so now we can produce transfer prepare & transfer fulfil without restarting admin with different piece of code.
http://{{BASE_URL_3001}}/test/producer/transfer/prepare
http://{{BASE_URL_3001}}/test/producer/transfer/fulfil
are now available in place of
http://{{BASE_URL_3001}}/test/producer